pyoozie/client.py

```diff
@@ -79,11 +79,14 @@ def __init__(self, url=None, user=None, timeout=None, verbose=True, **_):
         self._verbose = verbose  # Note: change default for verbose!
         self._stats = OozieClient.Stats()
         self._valid_server = False
+        if not type(session) == requests.sessions.Session: 
```

Comment:
Use `isinstance` in case there are subclasses of `Session`?

Reply:
You're too fast LOL pylint just yelled at me about this so fixing it now!